<p>Prefer the plain English versions of boolean operators, as they are the most accessible:</p>
<ul>
    <li>Use and instead of &&.</li>
    <li>Use or instead of ||.</li>
</ul>
<p>You may also use parentheses around boolean operators to clear any ambiguity. This can make long expressions easier to read.</p>
<h2>Noncompliant Code Example</h2>
<pre>
if foo && bar || baz:
    print("condition is true")
</pre>
<h2>Compliant Solution</h2>
<pre>
if (foo and bar) or baz:
    print("condition is true"
</pre>

