<p>Use snake_case to name functions.</p>
<p>Basically class_names start with a capital letter. If the Name is composed of multiple words, each word starts with a capital letter</p>

<h2>Noncompliant Code Example</h2>
<pre>
class_name characterbody3d
</pre>
<h2>Compliant Solution</h2>
<pre>
class_name CharacterBody3D
</pre>
