<p>Write constants with CONSTANT_CASE, that is to say in all caps with an underscore (_) to separate words:</p>
<p>This rule verifies that each function has at most the given number of parameters.</p>
<h2>Noncompliant Code Example</h2>
<pre>
const MAX_SPEED = 200
</pre>

