<p>Use snake_case to name files.</p>
<p>Basically filenames start with a small letter. If the Name is composed of multiple words, each word starts with a lower letter and is separated by a underscore</p>

<h2>Noncompliant Code Example</h2>
<pre>
PlayerBody.gd
</pre>
<h2>Compliant Solution</h2>
<pre>
player_body.gd
</pre>
