<p>Use lowercase for letters in hexadecimal numbers, as their lower height makes the number easier to read.</p>
<p>This rule verifies that each function has at most the given number of parameters.</p>
<h2>Noncompliant Code Example</h2>
<pre>
var hex_number = 0xFB8C0B
</pre>
<h2>Compliant Solution</h2>
<pre>
var hex_number = 0xfb8c0b
</pre>

