<p>Use snake_case to name functions.</p>
<p>Basically function names start with a small letter. If the Name is composed of multiple words, each word starts with a lower letter and is separated by a underscore</p>


<h2>Noncompliant Code Example</h2>
<pre>
func LoadLevel():
    pass
</pre>
<h2>Compliant Solution</h2>
<pre>
func load_level():
    pass
</pre>
