<p>Trailing commas are unnecessary in single-line lists, so don't add them in this case.</p>
<h2>Noncompliant Code Example</h2>
<pre>
enum Tiles {TILE_BRICK, TILE_FLOOR, TILE_SPIKE, TILE_TELEPORT}
</pre>
<h2>Compliant Solution</h2>
<pre>
enum Tiles {TILE_BRICK, TILE_FLOOR, TILE_SPIKE, TILE_TELEPORT,}
</pre>

