<p>Take advantage of GDScript's underscores in literals to make large numbers more readable.</p>
<h2>Noncompliant Code Example</h2>
<pre>
var large_number = 1234567890
var large_hex_number = 0xfffff8f80000
var large_bin_number = 0b110100101010
# Numbers lower than 1000000 generally don't need separators.
var small_number = 12_345
</pre>
<h2>Compliant Solution</h2>
<pre>
var large_number = 1_234_567_890
var large_hex_number = 0xffff_f8f8_0000
var large_bin_number = 0b1101_0010_1010
# Numbers lower than 1000000 generally don't need separators.
var small_number = 12345
</pre>

